src/main/java/sboot/example/dto/CommentDto.java

```diff
@@ -0,0 +1,17 @@
+package sboot.example.dto;
+
+import java.time.LocalDateTime;
+import lombok.Data;
+
+@Data
+public class CommentDto {
```

Comment:
are you sure that you need this dto?
you already have ReviewDto - you can create User, Product and Comment from it in your mappers